README.rst

```diff
@@ -1,448 +1,23 @@
-- ``--global-log``
-- ``--hypervisor-log-dir``
-- ``--shim-path``
-- ``--proxy-socket-path``
-
-
-Extensions
-~~~~~~~~~~
-
-list
-....
-
-The ``list`` command supports a "``--all``" option that provides
-additional information including details of the resources used by the
-virtual machine.
-
-Development
------------
-
-Follow the instructions in `Building`_, but you will also want to install:
-
-- doxygen
-- lcov
-- valgrind
-
-To build the API documentation::
-
-  $ doxygen Doxyfile
-
-Then, point your browser at ``/tmp/doxygen-cc-oci-runtime``. If you
-don't like that location, change the value of ``OUTPUT_DIRECTORY`` in
-the file ``Doxyfile``.
-
-Debugging
----------
-
-- Specify the ``--enable-debug`` configure option to the ``autogen.sh``
-  script which enable debug output, but also disable all compiler and
-  linker optimisations.
-
-- If you want to see the hypervisor boot messages, remove "`quiet`" from
-  the hypervisor command-line in "``hypervisor.args``".
-
-- Run with the "``--debug``" global option.
-
-- If you want to debug as a non-``root`` user, specify the "``--root``"
-  global option. For example::
-
-    $ gdb --args ./cc-oci-runtime \
-        --debug \
-        --root /tmp/cor/ \
-        --global-log /tmp/global.log \
-        --hypervisor-log-dir /tmp/ \
-        start --console $(tty) $container $bundle_path
-
-- Consult the global Log (see Logging_).
-
-Links
------
-
-.. _Intel: https://www.intel.com
-
-.. _`Clear Linux`: https://clearlinux.org/
-
-.. _`Qemu`: http://qemu.org
-
-.. _`qemu-2.7-lite`: https://github.com/01org/qemu-lite/tree/qemu-2.7-lite
-
-.. _OCI: https://www.opencontainers.org/
-
-.. _`cc-oci-runtime home page`: https://github.com/01org/cc-oci-runtime
-
-.. _runc: https://github.com/opencontainers/runc
-
-.. _`runc manpage`: https://github.com/opencontainers/runc/blob/master/man/runc.8.md`
+==================
 
-.. _Docker: https://github.com/docker/docker
+----
 
-.. _containerd: https://github.com/docker/containerd
+Project now in maintenance mode
+-------------------------------
 
-.. [#oci-spec]
-   https://github.com/opencontainers/runtime-spec
+.. warning::
 
-.. [#oci-runtime-cli]
-   https://github.com/opencontainers/runtime-spec/blob/master/runtime.md
+Now that the `next-generation runtime
+<https://github.com/clearcontainers/runtime>`_ has been `released
+<https://github.com/clearcontainers/runtime/releases/tag/3.0.0>`_, this
+project is now in maintenance mode.
 
-.. [#oci-vm-config-issue]
-   https://github.com/opencontainers/runtime-spec/pull/405
+All users are encouraged to switch to the new ``cc-runtime`` runtime.
+Installation and upgrade information is available on `the new runtime's wiki
+<https://github.com/clearcontainers/runtime/wiki>`_.
 
-.. [#oci-runtime-cli-clarification]
-   https://github.com/opencontainers/runtime-spec/issues/434
+To see the original project ``README``, click
+`here <https://github.com/01org/cc-oci-runtime/blob/master/README-original.rst>`_.
```

Comment:
nit: you should be able to make this a relative reference like [this](README-original.rst) (or the rst equiv?) I would think? Means it is maybe more future proof (and would de-ref correctly in your pending PR ;-)

Reply:
Agreed. Branch de-nitted! ;)